projects/dsp-ui/src/lib/viewer/views/resource-view/resource-view.component.ts

```diff
@@ -72,6 +73,10 @@ export class ResourceViewComponent implements OnInit, OnChanges, OnDestroy {
      */
     @Output() openProject: EventEmitter<ReadProject> = new EventEmitter<ReadProject>();
 
+    @Output() referredResourceClicked: EventEmitter<ReadLinkValue> = new EventEmitter<ReadLinkValue>();
+
+    @Output() referredResourceHovered: EventEmitter<ReadLinkValue> = new EventEmitter<ReadLinkValue>();
```

Comment:
Maybe we should rename the `openProject` output to a similar name as you did e.g. `referredProjectClicked`. We should keep it consistent and I like the name `referred...Clicked`. I will create a task. We shouldn't do it in this PR

Reply:
Done in DSP-1023

Reply:
If you think that `referred` is a bit too verbose we could also consistently change it (being aware of breaking the public API of course).

Reply:
No it's fine